### DISCUSSION TITLE
Scroll bar in Sub-menu

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1382] [Q&A]
In Getting Started document, it says:

There is a small scrollbar dot that appears along the right edge of the screen to indicate how far through the current list you are.
Additionally, this **scrollbar will blink rapidly when you are on the last item in a menu choice.**

- last item does not blink for me in any categories, on 2.19  (pinecil V1)   or  2.18 (V2)  - should this line be removed if it was changed to not blink? (while waiting for the blink, it changes to scroll the sentence detailing what the function is)

- In most sub-menus it is just a small dot on the right edge.  Only in the Power Settings does it appear to be made of 2 dots vertically.

**Comment by discip** [comment]
@River-b
1. It is not the item itself that is blinking, just the scrollbar! So please do not remove that line, since its content is true!

2. Sometimes the length of the scrollbar depends on the settings you choose. In some cases, an entry is hidden if you change certain settings.
In most cases, however, the scrollbar consists of at least 3 pixels.

**Reply by River-Mochi** [reply]
my scroll bar is not blinking at all. this is what I'm saying.  it is just  a Steady dot on the right , even when I am in the last item

**Comment by River-Mochi** [comment]
could it be that this Blinking does not work for any Pinecils?
 talking about the "dot"  not the words.

**Reply by discip** [reply]
@River-b
I don't know if we are talking about the same thing, but on my **Pinecil** the blinking works throughout its implementation, even with the latest release, flawlessly.

The scrollbar is meant only to blink if you reach the last value in a range.
e.g. For **`Motion sensitivity`** the scrollbar will blink when you reach **`9`**.

**Reply by River-Mochi** [reply]
yes that works for me, it is blinking at 9
I understand now.  it blinks  not for the  Sub-menu   names but the Elements inside  each submenu, if it is the last option, it blinks. 

I need to make the document more clear then, because right now the way it is written, it sounds like the sub-menu items should blink, but that is not exactly how this works.  Got it now !
